week-2-practise/level-3/5-indexOf.js

```diff
@@ -6,7 +6,9 @@
 // - return a new array with the replacement value inserted
 
 function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+   // complete this statement
```

Comment:
Okay, so this one has got a little bit messed up. There are some changes after "DO NOT EDIT BELOW THIS LINE" which are breaking things, but also the replace function is changing the original array.
You probably want to begin with
`var newArray = arr.slice();`
and work on the new array instead.